templates/v6.yml

```diff
@@ -1,9 +1,3 @@
-stages:
```

Comment:
why get rid of this?

Reply:
When I deleted it I didn't realize how they were being used downstream and thought they were redundant. I'll put them back!